dictionaries/CallMap_71_delta.php

```diff
@@ -40,6 +40,10 @@
     'session_gc' => ['int'],
     'timezone_identifiers_list' => ['list<string>|false', 'timezoneGroup='=>'int', 'countryCode='=>'?string'],
     'unpack' => ['array', 'format'=>'string', 'string'=>'string', 'offset='=>'int'],
+    'ReflectionFunction::getReturnType' => ['?ReflectionNamedType'],
+    'ReflectionFunctionAbstract::getReturnType' => ['?ReflectionNamedType'],
+    'ReflectionMethod::getReturnType' => ['?ReflectionNamedType'],
+    'ReflectionParameter::getType' => ['?ReflectionNamedType']
```

Comment:
To be consistent, these should go at the top between `DateTimeZone::listIdentifiers` and `SQLite3::createFunction`.